shared/constants/types/chat2/index.js

```diff
@@ -86,6 +86,7 @@ export type _State = {
   paymentConfirmInfo: ?PaymentConfirmInfo, // chat payment confirm screen data
   paymentStatusMap: I.Map<Wallet.PaymentID, Message.ChatPaymentInfo>,
   unsentTextMap: I.Map<Common.ConversationIDKey, ?HiddenString>,
+  flipStatusMap: I.Map<string, RPCChatTypes.UICoinFlipStatus>,
 } & TeamBuildingTypes.TeamBuildingSubState
```

Comment:
Should be an opaque type

Reply:
This type maps directly onto the component, there isn't any need for some other type here.